apps/extension/tests/mocks/mock-chrome.ts

```diff
@@ -1,18 +1,34 @@
 export const chrome = {
   storage: {
     local: {
-      clear() {},
-      get() {},
-      getBytesInUse() {},
-      onChanged() {},
-      remove() {},
-      set() {},
+      clear() {
+        // Mock implementation
```

Comment:
I assume these are to silence the no empty function rule? Really a nitpick, but `clear: noop`, is slightly better than a comment here.